src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
+      <section className='flex justify-center items-center h-screen w-screen flex-col'>
+        <h1 className=' text-5xl m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-5/6 h 3/6'></img>
```

Comment:
All of these images are basically decorative, so you don't need to write alt text, but you should still use `alt=''` on each one to indicate that it doesn't require alt text.